odoo/addons/base/tests/test_qweb.py

```diff
@@ -1762,6 +1702,977 @@ def test_space_remove_technical_all(self):
         rendered = self.env['ir.qweb']._render(view.id)
         self.assertEqual(str(rendered), result)
 
+class TestQwebCache(TransactionCase):
+    def test_01_render_xml_cache_base(self):
+        view1 = self.env['ir.ui.view'].create({
+            'name': "dummy",
+            'type': 'qweb',
+            'arch': """
+                <t t-name="base.dummy">
+                    <div t-cache="(cache_id,)" class="toto">
```

Comment:
Would you consider removing the `(` and `)` from the tests, so there are more examples or tuples without parentheses that facilitates inheritance. every little bits help.